core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.html

```diff
@@ -149,12 +149,12 @@
                     </div>
                     <div (keydown)="onMenuKeypress($event, 'learnMenu', {enter: ACTION_CLOSE})"
                          class="nav-item-bottom-link">
-                      <span class="underline">
+                      <a class="underline nav-item-bottom-link" tabindex="0">
```

Comment:
Hey @Rahul-lalwani-learner, the underline seems to be visible but would you confirm that when focus reaches the  `see all Lessons` element, on clicking enter, user is redirected to the appropriate page as well. If this does not occur, would please add this functionality as well and update the proof for that in the PR as well. Thanks!

Reply:
I have changed it as you asked me please let me know if there is something more on that.